src/app/views/query-runner/request/permissions/TabList.tsx

```diff
@@ -10,16 +10,20 @@ interface ITabList {
   classes: any;
   renderItemColumn: Function;
   renderDetailsHeader: Function;
+  permissionToken: boolean;
 }
 
-const TabList = ({ permissions, columns, classes, renderItemColumn, renderDetailsHeader }: ITabList) => {
+
+const TabList = ({ permissions, columns, classes, renderItemColumn, renderDetailsHeader, permissionToken }
```

Comment:
Using the [useSelector](https://react-redux.js.org/api/hooks#useselector) React Hook may help to reduce passing too many arguments into the TabList component since it has access to the redux state. This can be used to pick up whether the user is logged in or not.

Reply:
Ah okay perfect! Thanks for this!